jupyterlab_hdf/baseHandler.py

```diff
@@ -86,25 +78,16 @@ def initialize(self, notebook_dir):
     @gen.coroutine
     def get(self, path):
         """Based on an api request, get either the contents of a group or a
-        slice of a dataset and return it as serialized JSON.
+        selected hyperslab of a dataset and return it as serialized JSON.
         """
         uri = '/' + self.get_query_argument('uri').lstrip('/')
-        row = self.getQueryArguments('row', int)
-        col = self.getQueryArguments('col', int)
-
+        select = self.get_query_argument('select', default=None)
         try:
-            self.finish(json.dumps(self.manager.get(path, uri, row, col)))
-
+            self.finish(simplejson.dumps(self.manager.get(path, uri, select), ignore_nan=True))
```

Comment:
I trust that the reviver approach is probably the best option for handling this on the front end, but is there a reason to avoid handling it (or an option to) on the server?